sptx_format/test_experiment.py

```diff
@@ -3,6 +3,7 @@
 import pytest
 from pkg_resources import resource_filename
 
+import starfish  # noqa
```

Comment:
what's this for?

Reply:
There's an import recursion somewhere between `starfish` and `sptx_format` that shows up in the `sptx_format` tests if importing relatively without first importing `starfish`.